float32[] finger_positions
---
bool success
string message
